app/routes/administrative-units/administrative-unit/governing-bodies/index.js

```diff
@@ -0,0 +1,26 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class AdministrativeUnitsAdministrativeUnitGoverningBodiesIndexRoute extends Route {
+  @service store;
+
+  async model() {
+    let { id: administrativeUnitId } = this.paramsFor(
+      'administrative-units.administrative-unit'
+    );
+
+    let administrativeUnit = await this.store.findRecord(
+      'administrative-unit',
+      administrativeUnitId,
+      {
+        reload: true,
+        include: 'governing-bodies.has-time-specializations',
+      }
+    );
+
+    return {
+      administrativeUnit: administrativeUnit,
+      governingBody: await administrativeUnit.governingBodies.firstObject,
```

Comment:
I would add a comment about why only the first one is used. If I understood it correctly worship services related administrative units only have 1 governing body?

Reply:
Yes exactly, and then tthere are many gpverning bodies nested